MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,1069 @@
+
+
+
+## Date editor
+
+In the date editor, the DatePicker `Link` will be loaded.
+
+In `DatePicker`, the default date value is current date displayed by default. You can also adding nullable `DateTime` data type for the date picker property in data form, which allows you to set the current date and display current date in date editor. 
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    [DataType(DataType.Date)]
+    [Display(Name ="Birth Date")]
+    public DateTime? BirthDate { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+### Customizing format in date editor
+
+In the `DatePicker`, short date will be shown by default. You can change the applied format by setting the [Format]`Link`property in [DataFormDateItem]`Link`.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.GenerateDataFormItem += DataForm_GenerateDataFormItem;
+
+    private void DataForm_GenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {   
+
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Date")
+        {
+        (e.DataFormItem as DataFormDateItem).Format = "dd, MM, yyyy";
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+`Image`
+
+### Setting title for date editor
+
+You can set title to the date editor by handling the [Title]`Link`property in [DataFormDateItem]`Link`. The default value of the property is `Choose Date`.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.GenerateDataFormItem += DataForm_GenerateDataFormItem;
+
+    private void DataForm_GenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {   
+
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "BirthDate")
+        {       
+            (e.DataFormItem as DataFormDateItem).Title = "Select a date";
+        }
+    }
+{% endhighlight %}
+{% endtabs %}
+
+
+### Setting MaximumDate and MinimumDate in date editor
+
+You can customize the maximum and minimum allowable dates in the `DatePicker` by setting [MaximumDate]`Link` and [MinimumDate]`Link` in the [DataFormDateItem]`Link` respectively.
+
+{% tabs %}
+{% highlight MainPage.xaml.cs %}
+
+    dataForm.GenerateDataFormItem += DataForm_GenerateDataFormItem;
+
+    private void DataForm_GenerateDataFormItem(object sender, GenerateDataFormItemEventArgs e)
+    {
+        if (e.DataFormItem != null && e.DataFormItem.FieldName == "Date")
+        {
+            (e.DataFormItem as DataFormDateItem).MinimumDate = new DateTime(2017, 5, 5);
+            (e.DataFormItem as DataFormDateItem).MaximumDate = new DateTime(2017, 9, 2);
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+`DateMAximumImages`
+
+## Time editor
+
+In the time editor, the [TimePicker]`Link` will be loaded.
+
+
+In `TimePicker`, the default time value (12:00 AM) is displayed by default. You can also adding nullable `DateTime` data type for the time picker property in data form, which allows you to set default time and display the (12:00 AM) in time editor. 
```

Comment:
`DateTime` => TimeSpan, always mention TimeSpan for time editor

Reply:
property type added for time editor